sbt/docker-compose.yml

```
@@ -1,7 +1,7 @@
 version: '3'
 services:
   strongbox-web-integration-tests-sbt:
-    image: strongboxci/alpine:jdk8-mvn3.6-sbt1.3
+    image: strongboxci/alpine:jdk8-mvn3.6-sbt1.3.13
```

This doesn't need to be changed, because the sbt image is still tagged as `1.3`. Please revert this back :)

```suggestion
    image: strongboxci/alpine:jdk8-mvn3.6-sbt1.3
```

---

well. I did not want to change the original image, as it has 1.3.7